third_party/CMakeLists.txt

```diff
@@ -6,26 +6,38 @@
 
 # Might as well get used to double precision impact now as we'll want that eventually anyway
 # In Jolt
+
+# Suppress MSBuild output for all third party libraries
+if(WIN32 AND MSVC)
+    add_compile_options(/nologo)
+    string(APPEND CMAKE_CXX_FLAGS " /nologo")
+    string(APPEND CMAKE_C_FLAGS " /nologo")
+endif()
+
+# Force dynamic runtime for all libraries
+set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>DLL")
+
+# JoltPhysics configuration
 set(DOUBLE_PRECISION ON)
 
 if(THRIVE_AVX)
-  set(USE_AVX ON)
-  set(USE_AVX2 ON)
+    set(USE_AVX ON)
```

Comment:
Could you re-indent all the files using the existing formatting rules (two spaces per indentation level)?

Reply:
done :]